modules/local/presto/presto_splitseq.nf

```diff
@@ -0,0 +1,32 @@
+include { initOptions; saveFiles; getSoftwareName } from '../functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process PRESTO_SPLITSEQ {
+    tag "$meta.id"
+
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::presto=0.6.2=py_0" : null)              // Conda package
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/presto:0.6.2--py_0"  // Singularity image
+    } else {
+        container "quay.io/biocontainers/presto:0.6.2--py_0"                        // Docker image
+    }
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*_atleast-2.fasta"), emit: fasta
+    path("*_command_log.txt"), emit: logs
+
+    script:
+    """
+    SplitSeq.py group -s $reads --outname ${meta.id} -f CONSCOUNT --num 2 --fasta > "${meta.id}_command_log.txt"
+    """
+    // TODO: sed -n '1~4s/^@/>/p;2~4p' ${dedupped.baseName}_UMI_R1_R2_atleast-2.fastq > ${dedupped.baseName}_UMI_R1_R2_atleast-2.fasta
```

Comment:
Is this still needed?

Reply:
ah sorry, this is already fixed, as the `--fasta` option already generates the fasta file

Reply:
But removed the TODO now :)